lib/BuildSystem/BuildSystem.cpp

```diff
@@ -2205,17 +2205,12 @@ class ShellCommand : public ExternalCommand {
       }
       inheritEnv = value == "true";
     } else if (name == "working-directory") {
-#if __APPLE__
```

Comment:
Hmm, I wonder if it might be worth defining something like LLBUILD_SUBPROCESS_WORKING_DIRECTORY_SUPPORTED in Subprocess.h and using that here.  I think it is nicer for a UX perspective to let the user know it is unsupported as early as possible. It is conceivable that a client could have a manifest/build that does a substantial amount of work before the error in the actual spawn method is triggered.

Reply:
This is non-trivial and not something we expect that users would actually see outside of a tool development environment.

Reply:
Fair enough.